src/build.xml

```diff
@@ -5,14 +5,14 @@
 <PropertyGroup>
     <Solution>Dynamo.All.sln</Solution>
     <Platform>Any CPU</Platform>
-    <DotNet>net6.0</DotNet>
+    <DotNet>net8.0</DotNet>
```

Comment:
you can revert this and just run `msbuild build.xml /p:DotNet=net8.0` in the jenkins file to test it

Reply:
Yes, I will revert this before merge but leave this open for a while as a test branch for the BRE team to use.